scipy/optimize/_differentialevolution.py

```diff
@@ -123,17 +124,25 @@ def differential_evolution(func, bounds, args=(), strategy='best1bin',
         one of:
 
             - 'latinhypercube'
+            - 'sobol'
+            - 'halton'
             - 'random'
             - array specifying the initial population. The array should have
               shape ``(M, len(x))``, where M is the total population size and
               len(x) is the number of parameters.
               `init` is clipped to `bounds` before use.
 
         The default is 'latinhypercube'. Latin Hypercube sampling tries to
-        maximize coverage of the available parameter space. 'random'
-        initializes the population randomly - this has the drawback that
-        clustering can occur, preventing the whole of parameter space being
-        covered. Use of an array to specify a population subset could be used,
+        maximize coverage of the available parameter space.
+
+        'sobol' and 'halton' are superior alternatives and maximize even more
```

Comment:
It's not clear to me why they are superior alternatives. Can you briefly explain why, or give a reference? Use of the term superior should be justified.

Reply:
I will refer to `scipy.stats.qmc` for more details.